.devcontainer/ChibiOS-RP/Dockerfile.ChibiOS-RP.SRC

```
@@ -58,7 +58,7 @@ RUN mkdir -p /usr/local/bin/gcc
 # Clone ChibiOS main which has RP2040/RP2350 support
 # As soon as there is an official release that includes RP ports, this can be updated.
 # ============================================================
-RUN git clone --branch main https://github.com/chibios-upstream/chibios.git --depth 1 ./sources/ChibiOs
+RUN git clone --branch master https://github.com/chibios-upstream/chibios.git --depth 1 ./sources/ChibiOs
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)